server/gc/src/main/java/org/apache/accumulo/gc/GarbageCollectionAlgorithm.java

```diff
@@ -217,11 +222,30 @@ private void confirmDeletes(GarbageCollectionEnvironment gce,
         throw new RuntimeException(
             "Scanner over metadata table returned unexpected column : " + entry.getKey());
     }
+    Set<String> tableIdsAfter = gce.getTableIDs();
+    ensureAllTablesChecked(tableIdsBefore, tableIdsSeen, tableIdsAfter);
 
     confirmDeletesFromReplication(gce.getReplicationNeededIterator(),
         candidateMap.entrySet().iterator());
   }
 
+  @VisibleForTesting
+  protected void ensureAllTablesChecked(Set<String> tableIdsBefore, Set<String> tableIdsSeen,
```

Comment:
This is protected and marked vis for testing, but does not seem to be used in a test.  Is there a test change that was not checked in?

Reply:
Right, I need to add tests.